### ISSUE TITLE
[a11y] VerticalStep should have its state indicated in the a11y tree

### ISSUE THREAD

**Issue by JLou** [AxaFrance/design-system#1656]
# Accessibility verification of the component: `VerticalStep`

## Detected issues

There is no way to know the state of a `VerticalStep` right now. We have to rely on checking the content.
Adding text such as these in the title, hidden via CSS would help.
 
- Completed
- In progress
- Locked

These names should be overridable by our consummers
